web/components/reports/FcReportMetadata.vue

```diff
@@ -12,6 +13,22 @@
       </dd>
     </v-col>
   </v-row>
+
```

Comment:
Just a heads up, I've created a new component called FcCallout that I basically copied from this callout you created @spencermountain. I needed to use it in two places so it made sense. You should be able to use it here as well.